File: libs/toolkit-android/src/test/java/com/example/android/utils/BaseRobotTest.java
package com.example.android.utils;

import org.junit.After;
import org.junit.Before;
import org.junit.Rule;

import br.com.concretesolutions.requestmatcher.LocalTestRequestMatcherRule;
import br.com.concretesolutions.requestmatcher.RequestMatcherRule;

public abstract class BaseRobotTest {
  @Rule
  public final RequestMatcherRule server = new LocalTestRequestMatcherRule();

  @Before
  public void baseSetUp() {
    server.url("/");
  }

  @After
  public void baseTearDown() {
  }
}

File: libs/toolkit-android/src/test/kotlin/br/com/vitorsalgado/example/utils/BaseRobotTest.kt
package br.com.vitorsalgado.example.utils

import br.com.concretesolutions.requestmatcher.LocalTestRequestMatcherRule
import br.com.concretesolutions.requestmatcher.RequestMatcherRule
import org.junit.After
import org.junit.Before
import org.junit.Rule

abstract class BaseRobotTest {
  @Rule
  val server: RequestMatcherRule = LocalTestRequestMatcherRule()

  @Before
  fun baseSetUp() {
    server.url("/")
  }

  @After
  fun baseTearDown() {
  }
}
